CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # CHANGELOG
 
+### [v2.4 _(Jan 11, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.5)
+
+### ✨ Highlights
+- Introducing support Installment Payments (#148)
+
+### 🚀 Enhancements
+- Enable Place Order button on Internet Banking checkout option only if bank is selected. (#166)
+- Restrict Tesco Bill Payment to THB transactions only (#165)
+- Tesco Bill Payment added PRINT button (#164 )
+- Restrict internet banking payment method to THB orders only(PR #161)
+- Send Tesco barcode to customer's email.  (PR #158)
+- Make plugin compatible with PHP 7.2 (PR #168)
+
+### 👾 Bug Fixes
+- Fix for showing Tesco Bill Payment related information on checkout success page when using other payment methods (PR #163).
+- Fix for missing "Terms and Conditions" in Alipay and Tesco payment method. (PR #162 )
+- Fixed problem that translation to 'Select a card you want to proceed (PR #160)
+
+### [v2.4 _(Oct 2, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.4)
```

Comment:
@jacstn link is wrong. It's `https://github.com/omise/omise-magento/releases/tag/2.4`